### PR TITLE
log guice startup errors through monitor

### PR DESCRIPTION
**Pull request by holachuy**
Log guice startup errors through monitor. This helps with debugging errors at startup.

Sorry, not sure why the import order changes all of the sudden.

### CODE REVIEW

File: portability-api/src/main/java/org/datatransferproject/api/ApiMain.java [hunk 1]
```diff
@@ -167,17 +166,22 @@ public void initializeHttps(
...
+                  typeManager,
+                  cloudExtension.getJobStore(),
+                  keyGenerator,
+                  trustManagerFactory,
+                  keyManagerFactory,
+                  authServiceExtensions,
+                  tokenManager,
+                  extensionContext));
+    } catch (Exception e) {
+      monitor.info(() -> "Unable to initialize Guice in Apia", e);
```
Comment (bwillard) [inline portability-api/src/main/java/org/datatransferproject/api/ApiMain.java]: Why just info? Seems like this should be a warning.

Comment (bwillard) [inline portability-api/src/main/java/org/datatransferproject/api/ApiMain.java]: Apia?

Comment (bwillard) [inline portability-api/src/main/java/org/datatransferproject/api/ApiMain.java]: Also why not re-throw the exception as well?  Won't this lead to a NPE or some other hard to understand error latter?

File: portability-transfer/src/main/java/org/datatransferproject/transfer/WorkerMain.java [hunk 2]
```diff
@@ -123,17 +122,22 @@ public void initialize() {
...
+                  extensionContext,
+                  cloudExtension,
+                  transferExtensions,
+                  publicKeySerializers,
+                  decryptServices,
+                  symmetricKeyGenerator,
+                  asymmetricKeyGenerator,
+                  jobHooks));
+    } catch (Exception e) {
+      monitor.info(() -> "Unable to initialize Guice in Apia", e);
```
Comment (bwillard) [inline portability-transfer/src/main/java/org/datatransferproject/transfer/WorkerMain.java]: All the same comments here.

File: portability-api/src/main/java/org/datatransferproject/api/ApiMain.java (hunk 1 above)
Comment (holachuy) [reply portability-api/src/main/java/org/datatransferproject/api/ApiMain.java]: Done. Used severe.

Comment (holachuy) [reply portability-api/src/main/java/org/datatransferproject/api/ApiMain.java]: Done. Fixed typo.

Comment (holachuy) [reply portability-api/src/main/java/org/datatransferproject/api/ApiMain.java]: Done. Had it but lost it in copy and paste.

File: portability-transfer/src/main/java/org/datatransferproject/transfer/WorkerMain.java (hunk 2 above)
Comment (holachuy) [reply portability-transfer/src/main/java/org/datatransferproject/transfer/WorkerMain.java]: Done. Thx!

**Pull request merged**